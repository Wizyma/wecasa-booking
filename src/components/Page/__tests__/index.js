import React from 'react';
import { cleanup, render } from 'react-testing-library';

import WecasePage from '../';

afterEach(cleanup);

const { baseElement } = render(
  <WecasePage>
    <h1>Hello World</h1>
  </WecasePage>,
);

it('Page should match snapshot', () => {
  expect(baseElement).toMatchSnapshot();
});
