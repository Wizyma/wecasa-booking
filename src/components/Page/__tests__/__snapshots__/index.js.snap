// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should match snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bdVaJa gbxZqW"
    >
      <h1>
        Hello World
      </h1>
    </div>
  </div>
</body>
`;
